task.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:io';
+
+class Task {
+  int id;
+  String? title;
+  String? description;
+  bool isCompleted;
+
+  Task(this.id, this.title, this.description, this.isCompleted);
```

Comment:
use default constructor
```suggestion
  Task({
    this.id = 0,
    required this.title,
    required this.description,
    this.isCompleted = false,
  });
```